README.md

```diff
@@ -34,101 +62,64 @@ make deploy
 # validatingwebhookconfiguration.admissionregistration.k8s.io/capsule-validating-webhook-configuration created
 ```
 
-## Webhooks and CA Bundle
-
-Capsule is leveraging Kubernetes Multi-Tenant capabilities using the
-[Dynamic Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/),
-providing callbacks to add further validation or resource patching.
+Log verbosity of the Capsule controller can be increased by passing the `--zap-log-level` option with a value from `1` to `10` or the [basic keywords](https://godoc.org/go.uber.org/zap/zapcore#Level) although it is suggested to use the `--zap-devel` flag to get also stack traces.
 
-All this requests must be server via HTTPS and a CA must be provided to ensure that
-the API Server is communicating with the right client.
+## Admission Controllers
+Capsule implements Kubernetes multi-tenancy capabilities using a minimum set of standard [Admission Controllers](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) enabled on the Kubernetes APIs server: `--enable-admission-plugins=PodNodeSelector,LimitRanger,ResourceQuota`. In addition to these default controllers, Capsule implements its own set of Admission Controllers through the [Dynamic Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/), providing callbacks to add further validation or resource patching.
```

Comment:
requested admission controllers
`--enable-admission-plugins=PodNodeSelector,LimitRanger,ResourceQuota,MutatingAdmissionWebhook,ValidatingAdmissionWebhook`

Reply:
LGTM, just add them to your changes.

Reply:
LGTM, please add these changes.

Reply:
done